### PR TITLE
Change DIF to CDMX in mx_states

### PR DESCRIPTION
**Pull request by svmalvaez**
Thanks for your contribution!

A checklist is included below which helps us keep the code contributions
consistent and helps speed up the review process. You can add additional
commits to your pull request if you haven't met all of these points on your
first version. You'll be asked to rebase and squash the commits before the
final merge.

Please replace these instructions with a description of your change. The
'New Fields Only' checkboxes should be removed as well if your pull request
doesn't add any new fields.

**All Changes**
- [ ] Add an entry to the docs/changelog.rst describing the change.
- [ ] Add an entry for your name in the docs/authors.rst file if it's not
    already there.
- [ ] Adjust your imports to a standard form by running this command:
  
    `isort --recursive --line-width 120 localflavor tests`

**New Fields Only**
- [ ] Prefix the country code to all fields.
- [ ] Field names should be easily understood by developers from the target
    localflavor country. This means that English translations are usually
    not the best name unless it's for something standard like postal code,
    tax / VAT ID etc.
- [ ] Prefer '<country code>PostalCodeField' for postal codes as it's
    international English; ZipCode is a term specific to the United
    States postal system.
- [ ] Add meaningful tests. 100% test coverage is not required but all
    validation edge cases should be covered.
- [ ] Add `.. versionadded:: <next-version>` comment markers to new
    localflavors.
- [ ] Add documentation for all fields.

### CODE REVIEW

**Comment by benkonrath** [comment]
@svmalvaez Please edit the description of this PR to add a description of your change along with the relevant check box items.

File: localflavor/mx/mx_states.py [hunk 1]
```diff
@@ -9,11 +9,11 @@
     ('BCN', _('Baja California')),
     ('BCS', _('Baja California Sur')),
     ('CAM', _('Campeche')),
+    ('CDMX', _('Ciudad de México')),
```
Comment (benkonrath) [inline localflavor/mx/mx_states.py]: The state field is 3 characters long this won't work. You'll also need to modify the tests for this change.

Comment (jackboot7) [inline localflavor/mx/mx_states.py]: Would be using the three letter airport code for Ciudad de México (MEX) a fix for this?

Comment (sebasmagri) [inline localflavor/mx/mx_states.py]: @jackboot7 the official abbreviation is CDMX, so it would probably need the state field to be updated.

Comment (elyak123) [reply localflavor/mx/mx_states.py]: Also I believe it can be confusing with "Estado de México" which is a different state, I know its not very conventional but as a compromise could it be `CMX` ??
